amr-wind/equation_systems/icns/icns_ops.cpp

```diff
@@ -97,7 +112,13 @@ void advection_mac_project(FieldRepo& repo, const FieldState fstate, const bool
         mac::get_projection_bc(
             amrex::Orientation::high, bctype, repo.mesh().Geom()));
 
-    macproj.project(options.rel_tol, options.abs_tol);
+    if(has_overset){
+        auto phif = repo.create_scratch_field(1, 1, amr_wind::FieldLoc::CELL);
+        amr_wind::field_ops::copy(*phif, repo.get_field("p"), 0, 0, 1, 1);
```

Comment:
We need to average from node to cell-center here?

Reply:
fixed it